docs/syscall.md

```diff
@@ -0,0 +1,19 @@
+# System Calls
+
+## Experimental Display and Event System Calls
+
+These system calls are solely for the convenience of accessing the SDL library and are only intended for the presentation of RISC-V graphics applications. They are not present in the ABI interface of POSIX or Linux.
+
+### `draw_frame(screen, width, height)` - `0xbeef`
```

Comment:
Replace with following:
```
### `draw_frame` - Draw a frame around the SDL window

system call number: `0xbeef`
SYNOPSIS
void draw_frame(screen, width, height);
```

You can check Linux Programmer's Manual for reference.